src/uu/ls/src/ls.rs

```diff
@@ -2702,7 +2702,14 @@ fn file_is_executable(md: &Metadata) -> bool {
     // S_IXUSR -> user has execute permission
     // S_IXGRP -> group has execute permission
     // S_IXOTH -> other users have execute permission
-    md.mode() & ((S_IXUSR | S_IXGRP | S_IXOTH) as u32) != 0
+    #[cfg(all(
```

Comment:
This is quite heavy for a small gain.
Can we just disable the clippy warning for this line instead?

Reply:
sure